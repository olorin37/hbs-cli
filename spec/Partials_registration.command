#!/usr/bin/env bash
exec "$command_under_test" data.yaml template.hbs -r '**/*.hbs' -o "$output_file"
