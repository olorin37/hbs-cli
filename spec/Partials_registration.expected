Number: 1234567899.

A partial:
A => 3.14
B partial:
B => hbs,
C partial:
C => .
